helm/hpcc/templates/_helpers.tpl

```diff
@@ -590,6 +638,26 @@ A kludge to ensure mounted storage (e.g. for nfs, minikube or docker for desktop
       mountPath: {{ .volumePath | quote }}
 {{- end }}
 
+{{/*
+A kludge to ensure mounted storage (e.g. for nfs, minikube or docker for desktop) has correct permissions for PV
+*/}}
+{{- define "hpcc.changePlaneMountPerms" -}}
+{{- $storage := (.root.Values.storage | default dict) -}}
+{{- $planes := ($storage.planes | default list) -}}
+  {{- $includeLabels := .includeLabels | default list -}}
+{{- range $plane := $planes -}}
+ {{- if and ($plane.forcePermissions) (or ($plane.pvc) (hasKey $plane "storageClass")) -}}
+  {{- $mountpath := $plane.prefix -}}
+  {{- $matchedLabels := include "hpcc.doesStorageLabelsMatch" (dict "plane" $plane "includeLabels" $includeLabels) }}
+  {{- if ne $matchedLabels "" }}
```

Comment:
If you have to compare vs "" here it suggests the earlier code I flagged may be iffy!